src/core/library.c

```diff
@@ -313,6 +343,16 @@ MsQuicLibraryUninitialize(
     QUIC_TEL_ASSERT(MsQuicLib.ConnectionCount == 0);
 #endif
 
+#if DEBUG
+    uint64_t PerfCounters[QUIC_PERF_COUNTER_MAX];
+    QuicLibrarySumPerfCounters((uint8_t*)PerfCounters, sizeof(PerfCounters));
+    QUIC_DBG_ASSERT(PerfCounters[QUIC_PERF_COUNTER_CONN_ACTIVE] == 0);
```

Comment:
Let's put a comment along the lines of "All these are current active counts and should be zero on cleanup"